### DISCUSSION TITLE
Text generation from 3D images

### DISCUSSION THREAD

**Discussion by AceMcAwesome77** [Project-MONAI/MONAI#6764] [General]
Is there any MONAI framework model that can be used to generate text from 3D image volumes?  Similar to how some recent projects have generated reports from input XR images using transformers.  Thanks!

**Comment by KumoLiu** [comment]
Hi @AceMcAwesome77, thanks for your interest here!
We didn't currently have end-to-end text generation from 3d images. Free feel to create an issue or PR!
Thanks!
